Added the countervalue field for fees on operation details

juan-cortes opened the pull request:
![image](https://user-images.githubusercontent.com/4631227/54496912-d8ead700-48f4-11e9-88b9-dd604b7f6bfb.png)

🤷‍♂️

gre commented:
To be checked with design but I imagine it could be possible to put it next to it maybe in a greyish style in parenthesis. I also think we could try to turn on the "subMagnitude" to 3 (or something like that) so the thing is able to display 0.00001 USD in this extreme case.

juan-cortes commented:
This gives design something to work from, I don't mind _wasting_ the 2 minutes it took

juan-cortes commented:
- Increased the precision of the counter values
- Made the counter value use the rate of the operation date instead of the latest
- Updated the design with @dekkaki feedback
- Changed the default color for android on the values, it was getting `smoke` and wanted `darkblue` or similar, to match iOS.

![image](https://user-images.githubusercontent.com/4631227/54676150-4dde2c80-4b00-11e9-8866-07897cf730a8.png)
